salt/master.py

```diff
@@ -1452,7 +1452,7 @@ def _file_recv(self, load):
                 'file_recv_max_size limit of %d MB exceeded! %s will be '
                 'truncated. To successfully push this file, adjust '
                 'file_recv_max_size to an integer (in MB) large enough to '
-                'accommodate it.', file_recv_max_size, load['path']
+                'accommodate it.', self.opts['file_recv_max_size'], load['path']
```

Comment:
This fixes, but in a spaghetti way. Since patch is simple, I just opened another PR for this: https://github.com/saltstack/salt/pull/49638

Reply:
Update as your suggest. Thanks.

Reply:
Yoda expressions. :wink: But I already opened another PR.